### PR TITLE
N3600 support

### PR DESCRIPTION
**Pull request by saichint**
1. This PR is for adding support to N3600.
2. Event_history properties for bgp are now version independent
Tests run on n3k, n7k, n9k and n9k-f and they pass

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1062,17 +1062,17 @@ Manages configuration of a BGP instance.
 | N5k      | 7.3(0)N1(1)        | 1.2.0                  |
 | N6k      | 7.3(0)N1(1)        | 1.2.0                  |
 | N7k      | 7.3(0)D1(1)        | 1.2.0                  |
+| N9k-f    | 7.3(0)F3(2)        | 1.8.0                  |
```
Comment (mikewiebe) [inline README.md]: Was this something we missed?

File: README.md [hunk 2]
```diff
@@ -1133,22 +1133,22 @@ Enable/Disable the batching evaluation of prefix advertisements to all peers wit
 Enable/Disable enforces the neighbor autonomous system to be the first AS number listed in the AS path attribute for eBGP. Valid values are 'true', 'false', and 'default'. On NX-OS, this property is only supported in the global BGP context.
 
 ##### `event_history_cli`
-Enable/Disable/specify size of cli event history buffer. Valid values are 'true', 'false', 'size_small', 'size_medium', 'size_large', 'size_disable' and 'default'. Size can also be specified in bytes.
+Enable/Disable/specify size of cli event history buffer. Valid values are false', 'size_small', 'size_medium', 'size_large', 'size_disable'. Size can also be specified in bytes. Please Note: 'default' and 'true' values are deprecated in 1.8.0 versions and idempotency is not guranteed for this property due to platform differences.
```
Comment (mikewiebe) [inline README.md]: I would reword these notes a little bit.

`**Please Note:** Setting this value to 'default' or 'true' has been deprecated in module version 1.8.0.  This property is only used for BGP debugging purposes and idempotency is not guaranteed.`

File: README.md (hunk 1 above)
Comment (saichint) [reply README.md]: yes

**Comment by saichint** [comment]
Done with my changes. All tests pass, skips also checked and found ok.

File: README.md [hunk 3]
```diff
@@ -394,6 +394,7 @@ Platform | Description | Environments
 :--|:--|:--
 **N9k**   | Support includes all N9xxx models  | bash-shell, guestshell
```
Comment (mikewiebe) [inline README.md]: Please update the copyright at the end of the file.

Comment (saichint) [reply README.md]: Done

**Comment by mikewiebe** [comment]
👍  Will merge after Travis is happy.
